### DISCUSSION TITLE
How to Debug with Orchid

### DISCUSSION THREAD

**Discussion by lordjack** [orchidsoftware/platform#1692] [Q&A]
Hello everyone, I'm new with Orchid. You could give some tips on how to debug the code with the log :: debug ($ request-> Get ('Subject')) and log :: info ($ request-> get ('subject')). By using however, I do not know where I see these logs

I read this article that comments on the issue of logs, but does not show where we can see.
https://blog.orchid.software/how-to-write-code/

**Comment by tabuna** [comment]
Hi @lordjack. By default, logs are written to the following path `storage/logs`.  More information can be found [here](https://laravel.com/docs/logging). I also recommend using a [telescope](https://laravel.com/docs/8.x/telescope) or [clockwork](https://underground.works/clockwork/) when developing so that the presentation is more visual.
